com.unity.multiplayer.mlapi/Runtime/Profiling/PerformanceDataManager.cs

```diff
@@ -11,9 +12,10 @@ internal static class PerformanceDataManager
         internal static void BeginNewTick()
         {
             s_TickId = Math.Max(s_TickId, 0);
+            s_TickId++;
             s_ProfilerData = new PerformanceTickData
```

Comment:
you'll need to update this after the release->dev merge

Reply:
hey @becksebenius-unity & @kvassall-unity — we had release to develop merge recently, so what's up? :)

Reply:
Updated and rebased @MFatihMAR